src/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequest.cs

```diff
@@ -17,8 +17,7 @@ namespace Microsoft.Identity.Client.Internal.Broker
     internal class BrokerInteractiveRequest
     {
         internal Dictionary<string, string> _brokerPayload = new Dictionary<string, string>();
```

Comment:
Only private members are prefixed with _
Members should not be internal, we prefer exposing properties. When you expose these properties, try to make them read-only, e.g. 

```
internal IBroker Broker { get; } // you can assign to it only from the ctor
```